heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraContext.java

```diff
@@ -14,16 +14,40 @@
 
 package com.twitter.heron.scheduler.aurora;
 
+import java.io.File;
+
 import com.twitter.heron.spi.common.Config;
 import com.twitter.heron.spi.common.Context;
 
 public final class AuroraContext extends Context {
   public static final String JOB_LINK_TEMPLATE = "heron.scheduler.job.link.template";
+  public static final String JOB_TEMPLATE_ENV = "HERON_SCHEDULER_JOB_TEMPLATE";
+  public static final String JOB_TEMPLATE = "heron.scheduler.job.template";
+  public static final String JOB_TEMPLATE_DIR = "heron.scheduler.job.template.dir";
 
   private AuroraContext() {
   }
 
   public static String getJobLinkTemplate(Config config) {
     return config.getStringValue(JOB_LINK_TEMPLATE);
   }
+
```

Comment:
Instead of using environment variable, a better approach would be to check if these new configs are present, if these are present, use those - otherwise, default to what it is used to be. Alternatively, we can make this config mandatory - with the default pointing to heron.aurora file in the cluster directory.